homepage.css

```diff
@@ -116,14 +124,17 @@ ul li:hover ul{
 .content_1 p{
 	font-size:16px;
 	color:black;
+	font-family: 'Open Sans', sans-serif;
```

Comment:
10/09/2021: Changing the fonts style for the content_1(text that talk briefly about the music database) to be Open Sans, also the font weight.